### ISSUE TITLE
Dropdown buttons not unique

### ISSUE THREAD

**Issue by ccp84** [ccp84/dinogames_react#7]
During testing - dropdown buttons failed accessibility for not having unique id 
![image](https://github.com/ccp84/dinogames_react/assets/106958690/a8a6a662-a3cd-450a-819e-2af3f67004de)

**Comment by ccp84** [comment]
Set id of each to title, and added a key of the element id
